src/plugins/dashboard/public/dashboard_app/no_data/dashboard_app_no_data.tsx

```diff
@@ -37,9 +39,16 @@ export const DashboardAppNoDataPage = ({
       },
       application,
       http: { basePath },
+      customBranding: {
+        hasCustomBranding$: customBranding.customBranding.hasCustomBranding$,
+        showPlainSpinner: customBranding.customBranding.hasCustomBranding$,
+      },
     },
     dataViews,
     dataViewEditor,
+    customBranding: {
+      showPlainSpinner: useObservable(customBranding.customBranding.hasCustomBranding$),
```

Comment:
Remove this whole section

Reply:
right on - it's coming from core so doesn't need to be stated separately 👍🏻